packit_service/worker/build/copr_build.py

```diff
@@ -214,6 +228,15 @@ def run_copr_build(self) -> TaskResults:
             )
 
         for chroot in self.build_targets:
+            if chroot not in self.available_chroots:
+                self.report_status_to_all_for_chroot(
+                    state=CommitStatus.error,
+                    description="Not supported target",
```

Comment:
Just to make it really clear? (The test name can be a bit cluttered and mixed with dashes.)

```suggestion
                    description=f"Not supported target: {chroot}",
```